src/ccf/calc_ccf.jl

```diff
@@ -281,6 +281,38 @@ function ccf_1D(λ::A2, flux::A3, #line_list::ALL, #mask_shape1::A3
     return ccf_out
 end
 
+"""
+    `ccf_1D( λs, fluxes; ccf_plan )`
+    Compute the cross correlation functions of spectra with a mask.
+# Inputs:
+- `λs`: 1-d array of wavelengths
+- `fluxes`:  2-d array of fluxes, individual spectra along first dim
+# Optional Arguments:
+- `ccf_plan`:  parameters for computing ccf (BasicCCFPlan())
+# Returns:
+- 2-d array of size (length(calc_ccf_v_grid(plan)), size(flux, 2))
+"""
+function ccf_1D(λ::A2, flux::A3, #line_list::ALL, #mask_shape1::A3
+                plan::PlanT = BasicCCFPlan() ) where {
+                #; mask_shape::ACMS = TopHatCCFMask(), plan::PlanT = BasicCCFPlan() ) where {
```

Comment:
At this point, the CCFPlan approach is stable, so let's get rid of old comments where things were passed separately.  
I'm guessing you copied this from elsewhere, so maybe it's best to have a separate PR where we remove deprecated commented out code.

Reply:
I went ahead and cleaned up the code you copied from, so you can just clean these up here.

Reply:
Cleaned up.